src/rum/instrumentation/DdRumReactNavigationTracking.tsx

```diff
@@ -34,16 +34,16 @@ export default class DdRumReactNavigationTracking {
             return;
         }
 
-        if (this.registeredContainer != null && this.registeredContainer !== navigationRef) {
+        if (DdRumReactNavigationTracking.registeredContainer != null && this.registeredContainer !== navigationRef) {
```

Comment:
why did we need to replace `this` with `DdRumReactNavigationTracking`?
does that make any difference in a `static` function?

Reply:
Yes exactly, that's the point. The `this` in a static call won't actually store something you can read later.